erpnext_com/www/docs/user/manual/en/erpnext_integration/event_streaming.md

```diff
@@ -0,0 +1,120 @@
+
+For Example: If you have 4-5 companies hosted on different sites, one among them is the main site where you want to do ledger posting and other sites, where the Sales Invoices are generated. You could use Event Streaming in this case. For this, your child company sites can subscribe to the main company site for Item, Customer and Supplier Document Types and the main company can subscribe to the child companies for Sales Invoices.
+
+To access Event Streaming, go to:
+> Home > Integrations > Event Streaming
+
+## 1. Prerequisites
+Before creating an Event Producer, one same user needs to be created on both the sites which will be used to access the site and will act as an Event Subsciber. Make sure the user has the necessary permissions for creation, updation and deletion of the subscribed doctypes.
+
+## 2. How to set up Event Streaming
+
+### 2.1 Create an Event Producer
+1. The site which you want to subscribe to, is called as the Event Producer. Create an Event Producer document for the site you wish to get the updates from.
+2. Go to: **Home > Integrations > Event Streaming > Event Producer**.
+3. Enter the site URL of the site you want to subscribe to, in the Producer URL field.
+4. Add all the Document Types you want to subscribe to, in the Event Configuration child table.
+5. If you want to have the created documents with the same name as it is on the remote Event Producer site, check the 'Use Same Name' checkbox in the child table against the required Document Type.
+6. Set the Event Subscriber field to a user that will be used to create the documents fetched from the Event Producer. You need to create the same user both ways, i.e on the Event Consumer as well as the Event Producer site before creating the Event Producer.
+7. Save.
+8. The API key and API secret are then set in the Event Producer document for Authentication.
+
+    ![Event Producer](/docs/assets/img/setup/integrations/event-producer-doc.png)
+
+### 2.2 Approve Event Consumer on the Event Producer site
+1. After the Event Producer has been created, an Event Consumer automatically gets created on the other site. By default all the Subscribed Document Types have the status as 'Pending'. In order to enable the Event Consumer to consume the documents of these Document Types, their Status needs to be updated to 'Approved'.
+2. Go to: **Home > Integrations > Event Streaming > Event Consumer**.
+3. Once you open the Event Consumer document you will see all the Document Types that the consumer has subscribed to. Change the status from 'Pending' to 'Approved' for all the Document Types that you want to approve to be consumed. You can change the status to 'Rejected' if you do not want the documents of that Document Type to be consumed.
+4. Save.
+
+    ![Event Consumer](/docs/assets/img/setup/integrations/event-consumer-doc.png)
+
+**Note**: Document updates for Subscribed Document Types won't be synced unless they are Approved.
+
```

Comment:
add ### 2.3 Offline access with single site
add details about having a site on localhost and one on the cloud, so users can have 'offline ERP'.